rq/queue.py

```diff
@@ -196,6 +195,12 @@ def registry_cleaning_key(self):
         """Redis key used to indicate this queue has been cleaned."""
         return 'rq:clean_registries:%s' % self.name
 
+    @property
+    def scheduler_pid(self):
```

Comment:
Mind adding a return type annotation to this method?

Reply:
sure